rust/private/repository_utils.bzl

```diff
@@ -210,6 +210,7 @@ def BUILD_for_rust_toolchain(
         target_triple,
         include_rustc_srcs,
         default_edition,
+        rustfmt_version,
```

Comment:
nit: since the version is not actually used, can this be something like `include_rustfmt`?

Reply:
Changed to `has_rustfmt` :)